dbt/include/athena/macros/materializations/models/table/table.sql

```diff
@@ -98,7 +98,11 @@
         {%- endif -%}
 
         {% set query_result = safe_create_table_as(False, tmp_relation, compiled_code, language, force_batch) %}
-
+        {%- if language == 'python' -%}
```

Comment:
question: I think I faced some error when I tried to call the `safe_create_table_as` outside of the first call in the stack. Curious if you run into the same issue or if you handled it somehow?

Reply:
the issue i encountered is when a table already exists 
for high availability reasons the process is set up to create a temp table first and rename that to main table 
but the python code was like
1. rename temp table to main table
2. create temp table

and there will be error as temp table doesnt exist in the first step 

after this change:
1. create temp table
2. rename temp table to main table

Reply:
But other than that I didnt encounter any other issues